Merge to make maint on the pfsense documentation easier going forward (#9134)
netdata/netdata: packaging/installer/methods/pfsense.md

pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-yaml-5.2.txz
pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.20.0_3.txz
```

> **Note:** If you receive a `Not Found` error running any of the above commands [this bash script](./pfsense_getLatestPackages.sh) will update all of the above packages to the current ones in the repo.  Bash,grep,awk and curl are the only requirements.  These should all be pre-installed on Linux, MacOS, and Windows WSL.

Reviewer: ```suggestion

> If you receive a `Not Found` error running any of the above commands [this bash script](./pfsense_getLatestPackages.sh) will output updated commands based on the most up-to-date packages. Bash, grep, awk, and curl are the only requirements. These should all be pre-installed on Linux, MacOS, and Windows WSL.
```

Author: You are correct in your comments/corrections.  I think the wording is better now.
